_data/changelogs/component-identifier.yml

```diff
@@ -2,6 +2,23 @@ title: Identifier
 type: component
 changelogURL:
 items:
+  - date: NNN-NN-NN
+    summary: Updated Accessibility statement link guidance.
+    summaryAdditional: Updated the Identifier guidance to use the text "Accessibility statement" for the required link to an accessibility statement. We improved the supporting links that describe accessibility statements.
+    isBreaking: true
+    affectsAccessibility: true
+    affectsMarkup: true
+    githubPr: 2097
+    githubRepo: uswds-site
+    versionUswds: 3.5.0
+  - date: 2023-06-09
+    summary: Updated Accessibility statement link text. 
```

Comment:
These initially seem like duplicates. Do we want both? 

This second item is also missing an "Affects" tag

Reply:
Yes, this seems like a duplicate. I think we should use the second one, since this one only really affects content, not markup or accessibility

Reply:
I'll update

Reply:
Actually no. One sec

Reply:
The first one should focus just on the guidance changes. The second on the text changes top the component in support of the guidance changes